main.js

```diff
@@ -33,25 +43,18 @@ app.disableHardwareAcceleration();
 // The PSP will have a single instance. If an attempt to start a second instance is made,
 // the second one will be closed and the callback provided to `app.makeSingleInstance`
 // in the first instance will be triggered enabling it to show the PSP `BrowserWindow`.
-var appIsRunning = app.makeSingleInstance(function (commandLine) {
+app.on("second-instance", function (event, commandLine) {
```

Comment:
Question. Should this event listener be set up from the "gpii.app" component at the initialization process? 
@danailbd ?